src/screens/MainApp/DiagnoseRequest/MainScreen/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { View, Image } from 'react-native'
+import { ImageSelection, Text, Button } from '~/components'
+import styles from './styles'
+
+export const MainScreen = ({ navigation }) => {
+  const [showImageSelection, setShowImageSelection] = useState(false)
+
+  const onImagesSelected = (images) => {
+    const reduced = images.reduce((a, c) => {
+      a.push(c.path)
+      return a
+    }, [])
+    setShowImageSelection(false)
+    console.log(reduced)
+  }
+
+  return (
+    <>
+      {showImageSelection && <ImageSelection onCancel={() => setShowImageSelection(false)} onImagesSelected={onImagesSelected} />}
```

Comment:
This is acording to my style so it's not necessary to change, but I would put this as:

`{showImageSelection && (`  
``<ImageSelection``  
``onCancel={() => setShowImageSelection(false)}``   
``onImagesSelected={onImagesSelected}``  
``/>``  
``)}``